dask_ml/model_selection/_incremental.py

```diff
@@ -222,19 +222,20 @@ def get_futures(partial_fit_calls):
         _scores = {}
         _specs = {}
         for ident, k in instructions.items():
+            if not k:
+                continue
```

Comment:
I don't think this change is necessary. I need some time to clarify my thoughts on this.

Reply:
I've clarified my thoughts, and I'll try to implement them tonight. They are simple and put most of the responsibility for choosing a high performing model on `_adapt`.

There are two dimensions in `cv_results_["rank_test_score"]`: the final scores, and the time the model was scored at. This might change one of the tests:

``` diff
- assert search.cv_results_["rank_test_score"][search.best_index_] == 1
+ assert search.cv_results_["rank_test_score"][search.best_index_] <= 3  # 20 total models
```